src/wp-includes/option.php

```diff
@@ -1106,6 +1113,73 @@ function delete_option( $option ) {
 	return false;
 }
 
+/**
+ * Determine the appropriate autoload value based on input.
+ *
+ * This function checks the provided autoload value and returns a standardized
+ * value ('yes', 'no', 'default-yes', or 'default-no') based on specific conditions.
+ *
+ * @since 6.4.0
+ *
+ * @param mixed $autoload The autoload value to evaluate.
+ *
+ * @return string The standardized autoload value: 'yes', 'no', 'default-yes', or 'default-no'.
```

Comment:
@felixarntz I still don't love this naming. How about something like eager / lazy.

Reply:
I'm not a fan of the names either, but we should make sure the names we choose indicate that those are values assigned by default (i.e. not by a developer), and still keep the nuance between whether that default instructs to autoload or not.

`default-yes` and `default-no` sound weird, but I think they are quite clear given the other two values are `yes` and `no`. If we went with `eager` and `lazy` or with `default-eager` and `default-lazy`, that connection would not be as clear.